### PR TITLE
Remove more references to entity, entities, variable, var

### LINKED ISSUE #1270: Update code to replace references to entity/entities with Woodwork DataFrame
- Once the Entity object has been removed and replaced by a Woodwork DataFrame, variable names and parameters throughout the code should be updated to replace references to `entity` with new `dataframe` terminology. We should also search for locations where `variable` should be replaced by `column` or `var` by `col`.

- This issue will also involved **replacing entities to DataFrames**

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
### Remove more references to entity, entities, variable, var

Closes #1270 

Removes code references to entity, entities, variable and var and replaces them with dataframe, dataframes, column and col. 

Note, this update was only done on the python files. Documentation was not updated as those references should be fixed as part of other PRs.

### CODE REVIEW

File: featuretools/entityset/entityset.py [hunk 1]
```diff
@@ -819,8 +819,8 @@ def normalize_dataframe(self, base_dataframe_name, new_dataframe_name, index,
 
         base_dataframe_index = index
 
-        # --> Implementation: not sure that this is the same as using vtype Categorical because we can't just set a standard tag
-        # why did we set it to variable Categorical???
+        # --> Implementation: not sure that this is the same as using ltype Categorical because we can't just set a standard tag
+        # why did we set it to Categorical???
         # and why does it get reset when we did it above??
         # transfer_types[index] = ('Categorical', set())
```
Comment (tamargrey) [inline featuretools/entityset/entityset.py:825]: Also added these four lines that we should probably remove - they don't help with understanding, and I don't think we've found any reason to reset the index to be categorical (commented out like 825)

File: featuretools/computational_backends/feature_set.py [hunk 2]
```diff
@@ -70,9 +70,9 @@ def feature_trie(self):
         then used for all subsequent calls.
 
         The edges of the trie are RelationshipPaths and the values are tuples of
-        (bool, set[str], set[str]). The bool represents whether the full entity df is needed at
+        (bool, set[str], set[str]). The bool represents whether the full df is needed at
```
Comment (tamargrey) [inline featuretools/computational_backends/feature_set.py:73]: might be worth lengthening `df` to `dataframe` now that it doesn't have the extra `entity` descriptor

**Review by tamargrey — COMMENTED** [review]
Places that might still need updates references:

- line 635 in `feature_set_calculator` - "multivariable" - not sure if this is a variable reference
- feature_set_calculator.py - line 749 comment references `Id columns`
- feature_base.py - line 194 references the `Id return type`

These have to do with the `variable_types.utils` file that, if we aren't removing now, we might want to make an issue to remove later
- __init__.py line 3 `from . import variable_types`
- there's a todo in the utils file for determining if we're adapting graph_variable_types to woodwork - should either remove or make an issue
- test_variable_utils.py - only contains `test_list_variables` - could move to a different file - still has a bunch of commented out tests that can be removed if we're removing graph_variable_types

**Comment by thehomebrewnerd** [comment]
> Places that might still need updates references:
> 
> * line 635 in `feature_set_calculator` - "multivariable" - not sure if this is a variable reference
> * feature_set_calculator.py - line 749 comment references `Id columns`
> * feature_base.py - line 194 references the `Id return type`
> 
> These have to do with the `variable_types.utils` file that, if we aren't removing now, we might want to make an issue to remove later
> 
> * **init**.py line 3 `from . import variable_types`
> * there's a todo in the utils file for determining if we're adapting graph_variable_types to woodwork - should either remove or make an issue
> * test_variable_utils.py - only contains `test_list_variables` - could move to a different file - still has a bunch of commented out tests that can be removed if we're removing graph_variable_types

I made these updates and removed the code related to `graph_variable_types`. I created issue #1613 to document the need to remove the `list_variable_types` function at some point in the future, along with the associated tests.

File: featuretools/entityset/entityset.py [hunk 3]
```diff
@@ -342,11 +342,7 @@ def add_relationship(self,
 
         if parent_df.ww.index != parent_column:
             parent_df.ww.set_index(parent_column)
-        # Empty dataframes (as a result of accessing metadata)
-        # default to object dtypes for categorical columns, but
-        # indexes/foreign keys default to ints. In this case, we convert
-        # the empty column's type to int
```
[the remark this answers is not in the document]
Comment (thehomebrewnerd) [reply featuretools/entityset/entityset.py:348]: Added them back!

**Pull request merged**